packages/cli/test/me.endpoints.test.ts

```diff
@@ -0,0 +1,531 @@
+			const owner = await Db.collections.User!.findOneOrFail();
+			const ownerAgent = await utils.createAgent(app, owner);
+
+			for (const validPayload of VALID_PATCH_ME_PAYLOADS) {
+				const response = await ownerAgent.patch('/me').send(validPayload);
+
+				expect(response.statusCode).toBe(200);
+
+				const {
+					id,
+					email,
+					firstName,
+					lastName,
+					personalizationAnswers,
+					globalRole,
+					password,
+					resetPasswordToken,
+				} = response.body.data;
+
+				expect(validator.isUUID(id)).toBe(true);
+				expect(email).toBe(validPayload.email);
+				expect(firstName).toBe(validPayload.firstName);
+				expect(lastName).toBe(validPayload.lastName);
+				expect(personalizationAnswers).toBeNull();
+				expect(password).toBeUndefined();
+				expect(resetPasswordToken).toBeUndefined();
+				expect(globalRole.name).toBe('owner');
+				expect(globalRole.scope).toBe('global');
+			}
+		});
+
+		test('PATCH /me should fail with invalid inputs', async () => {
+			const owner = await Db.collections.User!.findOneOrFail();
+			const ownerAgent = await utils.createAgent(app, owner);
+
+			for (const invalidPayload of INVALID_PATCH_ME_PAYLOADS) {
+				const response = await ownerAgent.patch('/me').send(invalidPayload);
+				expect(response.statusCode).toBe(400);
+			}
+		});
+
+		test('PATCH /me/password should succeed with valid inputs', async () => {
+			const owner = await Db.collections.User!.findOneOrFail();
+			const ownerAgent = await utils.createAgent(app, owner);
+
+			const validPayloads = Array.from({ length: 3 }, () => ({
+				password: utils.randomValidPassword(),
+			}));
+
+			for (const validPayload of validPayloads) {
+				const response = await ownerAgent.patch('/me/password').send(validPayload);
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual(SUCCESS_RESPONSE_BODY);
+			}
+		});
+
+		test('PATCH /me/password should fail with invalid inputs', async () => {
+			const owner = await Db.collections.User!.findOneOrFail();
+			const ownerAgent = await utils.createAgent(app, owner);
+
+			const invalidPayloads: Array<{ password?: string }> = Array.from({ length: 3 }, () => ({
+				password: utils.randomInvalidPassword(),
+			}));
+
+			invalidPayloads.push({});
+
+			for (const invalidPayload of invalidPayloads) {
+				const response = await ownerAgent.patch('/me/password').send(invalidPayload);
+				expect(response.statusCode).toBe(400);
+			}
+		});
+
+		test('POST /me/survey should succeed with valid inputs', async () => {
+			const owner = await Db.collections.User!.findOneOrFail();
+			const ownerAgent = await utils.createAgent(app, owner);
+
+			const validPayloads = [SURVEY, {}];
+
+			for (const validPayload of validPayloads) {
+				const response = await ownerAgent.post('/me/survey').send(validPayload);
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual(SUCCESS_RESPONSE_BODY);
+			}
+		});
+	});
+
+	describe('Member requests', () => {
```

Comment:
ref previous question about difference `owner` and `member`

Reply:
See answer above.